soccer/src/soccer/planning/planner/collect_path_planner.cpp

```diff
@@ -110,6 +110,14 @@ Trajectory CollectPathPlanner::plan(const PlanRequest& plan_request) {
     std::vector<DynamicObstacle> dynamic_obstacles;
     fill_obstacles(plan_request, &static_obstacles, &dynamic_obstacles, false);
 
+    // Return an empty trajectory if the ball is hitting static obstacles
```

Comment:
Why is this still here?

Reply:
Can you clarify? It makes sense to do this if we happen to hit a static obstacle by chance. Right?